lib/contracts/custom-apps-contracts.ts

```diff
@@ -0,0 +1,11 @@
+import { SharedContracts } from './shared-contracts';
+
+export namespace CustomAppsContracts {
+    export interface ICustomAppContract {
+        name: string;
+        codename: string;
+        source_url: string;
+        config: string | null;
```

Comment:
config doesn't seem to be required, can't it be also undefined? 
<img width="845" alt="image" src="https://github.com/user-attachments/assets/bdf350a7-9067-48a1-b06f-eef95ee0dce6" />

Reply:
As far as my testing goes, no. Even when you don't set the `config` the property itself will always be present in the response and will either contain `null` or `string`. The type seems to be correct in this case.